src/client/utils/dom.js

```diff
@@ -520,8 +520,12 @@ export function isLocation (instance) {
     return instance && isObject(instance) && !isUndefined(instance.href) && !isUndefined(instance.assign);
 }
 
-export function isSVGElement (obj) {
-    return window.SVGElement && obj instanceof window.SVGElement;
+export function isSVGElement (instance) {
+    if (instance instanceof nativeMethods.svgElementClass)
+        return true;
+
+    return instance && isObject(instance) && !isUndefined(instance.ownerSVGElement) && instance.toString &&
```

Comment:
`inaccessibleTypeToStr` - ok
`isNullOrUndefine` - kinda ok (it has unnecessary `!obj` check)

all other - not ok